datasets/meldingen/meldingenbuurt/v1.0.0.json

```diff
@@ -0,0 +1,65 @@
+{
+    "id": "meldingbuurt",
```

Comment:
"meldingenBuurt" typo?

Reply:
Ik denk dat dit idd meldingenBuurt (ook in meervoud) moet zijn, ik pas het aan!